launch/launch/actions/reset_launch_configurations.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2021 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for the ResetLaunchConfigurations action."""
+
+from typing import Dict
+from typing import Optional
+
+from ..action import Action
+from ..launch_context import LaunchContext
+from ..some_substitutions_type import SomeSubstitutionsType
+from ..utilities import normalize_to_list_of_substitutions
+from ..utilities import perform_substitutions
+
+
+class ResetLaunchConfigurations(Action):
+    """
+    Action that resets launch configurations in the current context.
+
+    This action can be used to clear the launch configurations from the
+    context it was called in. It optionally can be given a dictionary
+    with launch configurations to be set after clearing. Launch
+    configurations given in the dictionary are evaluated before the
+    context launch configurations are cleared. This allows launch
+    configurations to be passed through the clearing of the context.
```

Comment:
nitpick: please use one sentence per line

> - [any] Each sentence must start on a new line.
>   - Rationale: For longer paragraphs a single change in the beginning makes the diff unreadable since it carries forward through the whole paragraph.

https://docs.ros.org/en/rolling/Contributing/Code-Style-Language-Versions.html#markdown-restructured-text-docblocks

Same as above.

Reply:
Fixed in [68b9ffe](https://github.com/ros2/launch/pull/515/commits/68b9ffeb6dff6ce532b2b297da781e44396201a7)